src/media/js/views/app/abuse.js

```diff
@@ -1,13 +1,13 @@
 define('views/app/abuse',
-       ['forms', 'l10n', 'notification', 'requests', 'urls', 'utils', 'z'],
-       function(forms, l10n, notification, requests, urls, utils, z) {
-
+    ['capabilities', 'forms', 'l10n', 'notification', 'nunjucks',  'requests',
+     'urls', 'utils', 'z'],
+    function(caps, forms, l10n, notification, nunjucks, requests, urls,
+             utils, z) {
     var gettext = l10n.gettext;
     var notify = notification.notification;
 
-    z.page.on('submit', '.abuse-form', function(e) {
-        e.preventDefault();
-        // Submit report abuse form
+    z.doc.on('submit', '.abuse-form', utils._pd(function(e) {
```

Comment:
the modal element lives in within body now since it isn't part of the main page.